types/tx/tx_wrapper_test.go

```diff
@@ -15,10 +14,10 @@ import (
 )
 
 var (
-	priv = ed25519.GenPrivKey()
-	pubkey = priv.PubKey().Bytes()
+	priv          = ed25519.GenPrivKey()
+	pubkey        = priv.PubKey()
 	pubKeyAddr, _ = sdk.Bech32ifyPubKey(sdk.Bech32PubKeyTypeAccPub, priv.PubKey())
-	addr = sdk.AccAddress(priv.PubKey().Address())
+	addr          = sdk.AccAddress(priv.PubKey().Address())
 )
```

Comment:
Can we please avoid global variables for test cases? Why? Because they tend to introduce race conditions that would then prevent/make very difficult tests parallelization. Please replace this `var` stanza with a function that returns all the desired keys/addresses

Reply:
Thanks @alessio